src/components/Common/Icon/Icon.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './Icon.css';
+
+import { getSvgs } from './iconHelpers';
+
+const iconsCollection = getSvgs();
+export interface IconProps {
+  name?: 'error' | 'visibility-off' | 'visibility-on'; //TODO: Destructure iconsCollection to support all available Icons, currently only supporting icons needed for log in
+  size?: '16' | '24' | '32' | '40';
+}
+
+const Icon = ({ size, name }: IconProps) => {
```

Comment:
I'd recommend using default values for props instead of defaultProps:

```suggestion
const Icon = ({ size = '24', name = 'error' }: IconProps) => {
```